capitals.py

```diff
@@ -0,0 +1,129 @@
+"""
+Makes a 10 questions long trivia game that asks what the capital of a country
+is and then gives three random capitals along with the right one in a
+randomized order.
+Every correct answer is 10 and every false answer is -10 points. Game doesn't
```

Comment:
Nevermind, I was confused because the score did not update instantly. Is there any way to update those scores right away?